packages/app/src/app/pages/Pro/index-v2.tsx

```diff
@@ -0,0 +1,343 @@
+import React from 'react';
```

Comment:
Took Sara's suggestion and created a new Page for v2

Reply:
🎉